website/src/content/docs/docs/introduction/editor/vscode.md

```diff
@@ -35,3 +62,31 @@ tsp code uninstall
 # For VSCode insiders
 tsp code uninstall --insiders
 ```
+
+## Telemetry
+
+The extension for Visual Studio Code collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](https://privacy.microsoft.com/privacystatement) to learn more. This extension respects the `telemetry.telemetryLevel` setting which you can find more information in the [FAQ](https://code.visualstudio.com/docs/supporting/faq#_how-to-disable-telemetry-reporting).
+
+### What telemetry data are collected
+
+- OperationTelemetry
+
+| Telemetry name    | Type         | Example         |
+| ---------------    | ----------------- | ---------------- |
+| EventName          | string | "start-extension" for example |
+| ActivityId         | string |  |
+| StartTime          | datatime    |  |
+| EndTime            | datatime    |  |
+| Result             | string    | "success","fail","cancelled", etc. |
+| LastStep           | string    |  |
+
+- OperationDetailTelemetry
+
+| Telemetry name    | Type         | Example         |
+| ---------------    | ----------------- | ---------------- |
+| ActivityId          | string |  |
+| EmitterName         | string | "@typespec/http-server-csharp" for example |
```

Comment:
this one would be good to have clarification that we only  include known emitter names

Reply:
Currently, this is no central registration of known emitters. That's why I can only take an example here.  But I can link to the emitters documented in TypeSpec.io

Reply:
We do right here, https://github.com/microsoft/typespec/blob/main/packages/typespec-vscode/src/vscode-cmd/emit-code/emitter.ts#L37-L38  but just talking about a general wording that only our emitters are recorded

Reply:
updated